posthog/hogql/database/database.py

```diff
@@ -592,6 +592,7 @@ def create_hogql_database(
         with timings.measure("select"):
             saved_queries = list(
                 DataWarehouseSavedQuery.objects.filter(team_id=team.pk)
+                .filter(managed_viewset__isnull=True)  # Ignore managed views for now
```

Comment:
You touched upon this on your loom very briefly, but why do we not want these in the sql editor UI? We just have a blank "managed views" section now, right?